server/models/List.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const listSchema = new mongoose.Schema({
+    name: {
+      type: String,
+      required: true,
+      unique: true
+    },
+    products: [{ 
+        type: Schema.ObjectId, 
+        ref: 'Product' }]
+  });
```

Comment:
I don't think this ref will work since the Product model was modelled as "products"

Reply:
fixed